process-services/1.11/install/container.md

```diff
@@ -0,0 +1,302 @@
+        ```
+
+2. Create a Kubernetes secret for the Process Services license file with the following command:
+
+    ```bash
+    kubectl create secret generic licenseaps --from-file=./activiti.lic
+    --namespace=$NAMESPACE
+    ```
+
+3. Add the Alfresco Kubernetes repository to Helm with the following command:
+
+    ```bash
+    helm repo add alfresco-stable https://kubernetes-charts.alfresco.com/stable
+    ```
+
+4. [Update the properties](#helm-properties) for the Process Services chart.
+
+5. **(Optional)**: To enable the Identity Service:
+
+    1. Enable the Identity Service in the `alfresco-infrastructure` section of the `values.yaml`:
+
+        ```text
+        alfresco-identity-service:
+        enabled: true
+        ```
+
+    2. Set the Process Services environment variable `IDENTITY_SERVICE_ENABLED` to `true`.
+
+    3. Set the Process Services environment variable `IDENTITY_SERVICE_AUTH` to `http://$DNS/auth`.
+
+    4. Ensure the Identity Service settings for Process Workspace are set in the [properties](#helm-properties).
+
+6. Deploy the chart using a command similar to the following:
+
+    ```bash
+    helm install alfresco-stable/alfresco-process-services --set dnsaddress="http://$DNS" --namespace=$NAMESPACE --set license.secretName=licenseaps
+    ```
+
+The applications will be available at the following URLs:
+
+* Process Services: `http://$DNS/activiti-app`
+* Process Services Administrator: `http://$DNS/activiti-admin`
+* Process Workspace: `http://$DNS/`
+* Identity Service administrator console: `http://$DNS/auth/admin`
+
+>**Note:** To change the context paths of any of the applications, edit the ingress paths:
+
+```yaml
+ingress:
+    path: /activiti-app
+```
+
+### Helm properties
+
+The following information details the properties that can be set for Process Services when deploying via Helm on Amazon's Elastic Container Service for Kubernetes (Amazon EKS).
+
+The following properties can be configured in the `values.yaml` file or overridden as environment variables:
+
+|Property|Description|
+|--------|-----------|
+|ACTIVITI_DATASOURCE_DRIVER|The JDBC driver used to connect to the database. The default is `org.postgresql.Driver`. |
+|ACTIVITI_HIBERNATE_DIALECT|The dialect that Hibernate uses that is specific to the database type. The default is `org.hibernate.dialect.PostgreSQLDialect`. |
+|ACTIVITI_LICENSE_MULTI_TENANT|Set whether the license used is a multi-tenant one or not. The default is `false`. |
+|ACTIVITI_DATASOURCE_URL|The location of the database that will be used. |
+|ACTIVITI_DATASOURCE_USERNAME|The username to access the database with. The default is `alfresco`. |
+|ACTIVITI_DATASOURCE_PASSWORD|The password for the `ACTIVITI_DATASOURCE_USERNAME` user. The default is `alfresco`. |
+|ACTIVITI_CORS_ENABLED|Sets whether Cross Origin Resource Sharing (CORS) is enabled or not. The default is `true`. |
+|ACTIVITI_CORS_ALLOWED_ORIGINS|The host origins allowed in CORS requests. The default is `*`. |
```

Comment:
`*` can no longer used as the value for this property. By default it's not set. A new property ACTIVITI_CORS_ALLOWED_ORIGIN_PATTERNS should be used in case `*` needs to be used.

Reply:
This change is for 2.x.

The default is not set and cannot be * ever.

The description will be the host origins patters allowed in CORS requests. the default is * but can also be a pattern.

Reply:
done